distribution/bwc/build.gradle

```diff
@@ -292,7 +292,12 @@ BuildParams.bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInf
 
       createBuildBwcTask(projectName, "${baseDir}/${projectName}", projectArtifact)
     }
-
+    // Create build tasks for a task-cancellation test plugin used for compatibility testing
+    if (bwcVersion.onOrAfter(Version.fromString("8.0.0"))) {
```

Comment:
This is temporary, yes? We shouldn't need to restrict versions here as this should only exist in branch for which the current snapshot branches also have this project.